### ISSUE TITLE
Finalize the password / login feature.

### ISSUE THREAD

**Issue by JacquelineCasey** [jackcasey067/flask-tutorial#1]
There are a few more things to add to the feature:
  - Write up what I have learned in the README
  - Generate a better key and password, stored in .env
  - Move sent HTML to separate HTML Files.
  - Update the homepage HTML with links to new pages.

**Comment by JacquelineCasey** [comment]
Did some work:
- Separated HTML from the login function.
   - I am considering adding a page to the logout function that provides a button that actually triggers the logout.
- Updated the homepage with new links.

**Comment by JacquelineCasey** [comment]
I updated the README, and I fixed up security loopholes (not that it matters, I publically broadcast the password on the homepage).

---

I did not end up updating the login endpoint with its own page. It might be better to do that when I learn templates, so that various pages (all pages?) could contain info regarding if you are or are not logged in, or if it is even possible to log out on the logout page. (See #3)

Also, many apps do not have a dedicated logout page. I am maybe looking for something like a loading image that plays when needed: https://stackoverflow.com/questions/14525029/display-a-loading-message-while-a-time-consuming-function-is-executed-in-flask.

---

Regardless, this issue is closed.
